ocaml/idl/sdk-README.html

```diff
@@ -1,110 +0,0 @@
-<html>
```

Comment:
Also the other html files in ocaml/idl: do they have replacements somewhere? Or is that not needed?